<launch>
    <node name="rule_based_controller" pkg="pamaral_decision_making" type="rule_based_controller.py" output="screen">
        <param name="quaternion_poses" value="$(find pamaral_decision_making)/config/quaternion_poses/positions.json"/>
        <param name="rules_path" value="$(find pamaral_decision_making)/config/rules.json"/>
    </node>
</launch>